cli/flox/src/commands/init/node.rs

```diff
@@ -3,37 +3,65 @@ use std::path::Path;
 
 use anyhow::{anyhow, Result};
 use flox_rust_sdk::flox::Flox;
+use flox_rust_sdk::models::environment::global_manifest_path;
 use flox_rust_sdk::models::environment::path_environment::InitCustomization;
-use flox_rust_sdk::models::environment::{global_manifest_lockfile_path, global_manifest_path};
 use flox_rust_sdk::models::lockfile::LockedManifest;
 use flox_rust_sdk::models::manifest::PackageToInstall;
 use flox_rust_sdk::models::search::{do_search, PathOrJson, Query, SearchParams, SearchResult};
-use indoc::formatdoc;
+use indoc::{formatdoc, indoc};
 use log::debug;
 use semver::VersionReq;
 
 use super::{format_customization, InitHook, AUTO_SETUP_HINT};
 use crate::config::features::Features;
 use crate::utils::dialog::{Dialog, Select};
 
+const NPM_HOOK: &str = indoc! {r#"
+                # Install nodejs depedencies
+                npm install"#};
```

Comment:
Auto format isn't very opinionated for these. Happy to do something different if you have a preference. Right now there's a newline at the start but not the end, and if I change that I'll have to tweak newlines elsewhere

Reply:
Ah its not supposed to be 
```
# Install nodejs depedencies npm install
```

because this is part of the shell script... my bad disregard this question then :D